src/CollAction/Views/Projects/Create.cshtml

```diff
@@ -2,17 +2,17 @@
 
 @inject IViewLocalizer Localizer
 
-@model CollAction.Models.CreateProjectViewModel
```

Comment:
Right now CreateProject and EditProject use the same data. I can imagine it'd be temping to unify the viewmodels, but this feels rather premature.

I'm almost certain that in the future we'll do things slightly different. A project that's being editted for example:
* Might not be allowed to change the start date retroactively
* Might be able to administrate comments, block unwanted users (just fictional examples). Stuff that's not relevant to CreateProject

Tying them together means changing something in the edit workflow, might break create, and retrospectively.

Trying to unify stuff like this should be done at the end of the project, when we know more about how the final project looks.